topology/src/main/java/io/camunda/zeebe/topology/api/TopologyManagementRequests.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.topology.api;
+
+import io.atomix.cluster.MemberId;
+import java.util.Set;
+
+/** Defines the supported requests for the topology management. */
+public final class TopologyManagementRequests {
```

Comment:
:wrench: IMO this is not needed, we can just use the types defined in the `.proto` directly.